common/src/main/java/org/apache/hertzbeat/common/constants/ConfigConstants.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hertzbeat.common.constants;
+
+/**
+ * Extract all public strings that the project needs to configure in yml.
+ */
+
+public interface ConfigConstants {
+
+    /**
+     * Package name constant.
+     */
+    interface PkgConstant {
+        String PKG = "org.apache.hertzbeat";
+    }
+
+    /**
+     * hertzbeat project module constant.
+     */
+    interface FunctionModuleConstants {
+
+        String ALERT = "alerter";
```

Comment:
It is better to create a new constant for alerter here, because there are two place use this constant. One of them was **alert** before and the other was **alerter**.

Reply:
get

Reply:
done